redis/src/main/scala/zio/redis/options/Shared.scala

```diff
@@ -4,14 +4,18 @@ trait Shared {
   sealed trait Update { self =>
     private[redis] final def stringify: String =
       self match {
-        case Update.SetExisting => "XX"
-        case Update.SetNew      => "NX"
+        case Update.SetExisting    => "XX"
+        case Update.SetNew         => "NX"
+        case Update.SetLessThan    => "LT"
+        case Update.SetGreaterThan => "GT"
```

Comment:
These are independent from XX and NX according to the specification, i.e. we'll need to be able to express both.

Reply:
Ah, scratch that, the docs are misleading.